tensorflow/contrib/seq2seq/python/ops/attention_wrapper.py

```diff
@@ -1077,6 +1077,7 @@ def __init__(self,
                cell,
                attention_mechanism,
                attention_layer_size=None,
+               attention_layer=None,
```

Comment:
Can you move this towards the end of the function signature to avoid breaking existing users?